### PR TITLE
Added more actions

### LINKED ISSUE #27: [Proposal] Implement missing actions
### Proposal 

Implement the missing actions.

### Motivation

Some actions cannot be performed, preventing some environments from being solved:
- Dragging
- Scroll wheel
- Emit keys (e.g., Ctrl+C)

### Pitch

Implement such actions in Selenium.

### Alternatives

Most of these actions look doable but could run into issues:
- Getting observation in the middle of a mouse drag might cause issues.
- Scroll wheel needs an arbitrary scroll offset.
- Clipboard might not work or might interfere with the system clipboard (since we're not sandboxing the environment).

There are probably hacks for these issues (e.g., replace clipboard copy/paste with JS calls). The worst case scenario is that they cannot be done, in which case we might need to consider moving away from Selenium.

### Checklist

- [x] I have checked that there is no similar [issue](https://github.com/Farama-Foundation/miniwob-plusplus/issues) in the repo (**required**)

### PR DESCRIPTION
**Pull request by ppasupat**
# Description

Implemented the missing actions, including:
* Mouse actions at the specified coordinates (move, double-click, mousedown, mouseup).
* Scroll wheel (scroll up, scroll down). The scroll amount is specified in the config).
* Key press, including special keys like "`<PageUp>`" and combinations like "Ctrl+C". The allowlist of key combinations is specified in the config.
* Field typing, which directly types the text from a task field (provided it is entirely in ASCII).

The tests and documentation were updated accordingly.

Fixes #27.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `pre-commit run --all-files` (see `CONTRIBUTING.md` instructions to set it up)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/content/key_combinations.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Key Combinations
+
+**TODO:** Write this.
```
Comment (pseudo-rnd-thoughts) [inline docs/content/key_combinations.md:3]: Will be completed in a separate PR?

**Review by pseudo-rnd-thoughts — APPROVED** [review]
This looks really good and impressive. 
Im reminded that after we have completed all of the updates that you are interested are doing, we should reconfirm performance to check that we haven't broken something along the way if this is possible

File: docs/content/key_combinations.md (hunk 1 above)
Comment (ppasupat) [reply docs/content/key_combinations.md:3]: Yes, I will work on cleaning up the doc + README next.
